modules/xmpp/JingleSessionPC.js

```diff
@@ -1999,24 +1999,25 @@ export default class JingleSessionPC extends JingleSession {
                     ? tpc.removeTrackMute.bind(tpc, track)
                     : tpc.addTrackUnmute.bind(tpc, track);
 
-            if (!tpcOperation()) {
-                finishedCallback(`${operationName} failed!`);
+            tpcOperation()
+                .then(shouldRenegotiate => {
+                    let promise = Promise.resolve();
```

Comment:
You  don't need this var here. Just call the `finishedCallback` if  no renegotiation  is needed

Reply:
ok